front/src/styles/scss/_uiManchette.scss

```diff
@@ -1 +1,2 @@
 @import '@osrd-project/ui-manchette/dist/theme.css';
+  
```

Comment:
what’s this madness

Reply:
I guess it doesn’t want a space on the newline?

Reply:
done